package.json

```diff
@@ -80,10 +84,20 @@
     ],
     "topics": {
       "force": {
+        "description": "commands to create salesforce metadata",
```

Comment:
@iowillhoit - let me know if this work for topics/subtopics! They were manually generated, so I'm sure are prone to user-error here.